c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -1,16 +1,19 @@
 #include <arpa/inet.h>
 #include <iostream>
+#include <regex> // std::regex
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <strings.h>
+#include <vector>
```

Comment:
Looks like this header isn't used?
```suggestion
```

Reply:
I removed this as part of [`ae81dca` (#227)](https://github.com/fastly/js-compute-runtime/pull/227/commits/ae81dca453df75f519a3af68046348497954ab0e)